src/github/copilotApi.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import fetch from 'cross-fetch';
```

Comment:
any reason this library over just `fetch`?

Reply:
I think we use this to work serverless.

Reply:
Maybe a historic thing then since [fetch is everywhere](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) now?